scripts/config/webpack/util/env.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable prefer-import/prefer-import-over-require */
+
+const urlJoin = require('proper-url-join');
```

Comment:
You have removed all the validation that was being done. Could you restore it? Additionally, because of that, we won't need `proper-url-join`.